polkadot/node/network/statement-distribution/src/v2/requests.rs

```diff
@@ -748,6 +752,10 @@ fn validate_complete_response(
 						rep_changes.push((requested_peer, COST_UNREQUESTED_RESPONSE_STATEMENT));
 						continue
 					}
+
+					if disabled_mask.get(i).map_or(false, |x| *x) {
+						continue
+					}
```

Comment:
nit: This and the check at line 741 look the same. Can they be moved out of the `match`?